verify_stage_sigs

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+. settings
+
+ARCHES="x86_64 source"
```

Comment:
Should we move this into `settings` as a new global variable that `sign_stage_rpms` then also uses?

Reply:
probably. `build_stage_repository` also uses arches, but doesn't load settings (and uses them slightly differently too)